### ISSUE TITLE
Can't open the Pvalue file!

### ISSUE THREAD

**Issue by biozzq** [Menggg/BLINK#3]
Dear all,

I am performing GWAS with the C version of Blink and  I'm getting the following error message.

**************************
*   BLINK v0.01 start   *
**************************
****************************************************************
*   BLINK is written by Meng Huang and designed by Zhiwu Zhang  *
*  Please visit zzlab.net to get more information about BLINK  *
****************************************************************

Input arguments:   --file auto --plink --gwas --parallel 6 --cycle_size 2000 --trait 2 --out TTN
one cycle include 2000 SNPs inside!

The number of traits = 1
--------
In PLINK files
number of markers is 19824366
number of individuals is 17959
--------
plink_mode is 1

The number of traits = 8

The number of marker = 19824366
--------
The number of individual = 17959
The number of marker = 19824366
--------
The number of covariates = 10

=====================================
na_length=0
===TTN===
Can't open the Pvalue file!
All the time consumption = 6521 seconds

Any suggestions will be appreciated. Thank you.

Zheng zhuqing

**Comment by hejunyi-forever** [comment]
I have the same trouble as you, did you find out the reason? @biozzq

**Comment by p-polacek** [comment]
If you're still looking for a solution, it might be that your phenotype names have spaces in them. Replace them with underscores for example.

**Comment by Lamoumni18** [comment]
Hello everyone, the same problem here, any solutions?

**Comment by p-polacek** [comment]
@Lamoumni18 did my suggestion above not fix it?

**Comment by Lamoumni18** [comment]
Nope, but through some investigation, I found out that the problem is related to covariates: The same set analysis working with 6 PCs, result in the mentioned error when working with 10 PCs or more.

**Comment by Neo-xbx-00** [comment]
I have try many times but still recieve this error. I make sure that my phenotype names do not have spaces in them. And I only used 3 PCs.
